tests/Unit/Model/RockTheVoteReportTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Tests\Http;
+
+use ErrorException;
+use Tests\TestCase;
+use Chompy\Models\RockTheVoteReport;
+
+class RockTheVoteReportTest extends TestCase
+{
+    /**
+     * Test that expected fields are parsed from given API response object.
+     *
+     * @return void
+     */
+    public function testCreateFromApiResponseWithValidResponse()
+    {
+        $apiResponse = (object) [
+            'status'=> 'queued',
+            'status_url' => 'https://register.rockthevote.com/api/v4/registrant_reports/17',
+        ];
+        $since = '2019-12-19 00:00:00';
+        $before = '2020-02-19 00:00:00';
+
+        $report = RockTheVoteReport::createFromApiResponse($apiResponse, $since, $before);
+
+        $this->assertEquals($report->id, 17);
+        $this->assertEquals($report->since, $since);
+        $this->assertEquals($report->before, $before);
+        $this->assertEquals($report->status, 'queued');
+        $this->assertEquals($report->user_id, null);
+        $this->assertDatabaseHas('rock_the_vote_reports', [
+            'id' => 17,
+        ]);
```

Comment:
This seems redundant since you're testing the same thing against the Eloquent model directly above.